.ddev/config.yaml

```diff
@@ -1,23 +1,22 @@
+name: mautic
```

Comment:
I guess we don't need the `name` here. Otherwise it will break the setup for people who cloned the project in a different folder.

```suggestion
```